modules/nw-sriov-configuring-operator.adoc

```diff
@@ -59,32 +49,34 @@ Follow this procedure to disable it.
 * You must have installed the SR-IOV Operator.
 
 .Procedure
-. Update the default SriovOperatorConfig Custom Resource
+
+. To update the `default` SriovOperatorConfig CR, enter the following command:
 +
 ----
-$ oc patch sriovoperatorconfig default --patch '{ "spec": { "enableInjector": false}}' --type=merge -n openshift-sriov-network-operator
+$ oc patch sriovoperatorconfig default \
+  --type=merge -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableInjector": false } }'
 ----
 
-. To re-enable Network Resources Injector
+. To reenable the Network Resources Injector, enter the following command:
 +
 ----
-$ oc patch sriovoperatorconfig default --patch '{ "spec": { "enableInjector": true}}' --type=merge -n openshift-sriov-network-operator
+$ oc patch sriovoperatorconfig default \
```

Comment:
This procedure is confusing to me. At the end of the procedure, is the Network Resources Injector disabled, as the procedure heading implies? Or is the second step actually enabling it after step 1 disabled it? If it's the latter, then I don't think it should be a step in this procedure. It should either be its own, or mentioned after the procedure somehow.

Reply:
True. I consolidated it into the same procedure.